package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@leapfrogtechnology/sync-db",
   "description": "Command line utility to synchronize and version control relational database objects across databases",
-  "version": "1.0.0-beta.11",
+  "version": "1.0.0-beta.12",
```

Comment:
Let's release separately. Not in the same PR. @cham11ng @aanchal-adk @samirsilwal

Reply:
We are abusing this ability.